airflow/dag_processing/manager.py

```diff
@@ -303,17 +302,18 @@ def _run_parsing_loop(self):
 
             self._kill_timed_out_processors()
 
-            self._refresh_dag_bundles()
+            known_files: dict[str, set[DagFileInfo]] = {}
```

Comment:
We don't want to reset `known_files` every loop - we may not refresh every bundle every loop! Need to move this out of the while.

Reply:
oh yeah yup